data-prepper-plugins/opensearch/src/test/java/org/opensearch/dataprepper/plugins/sink/opensearch/s3/S3FileReaderTest.java

```diff
@@ -85,12 +78,18 @@ void readFile_with_invalidPath_throws_parsing_exception() {
     void readFile_with_bigger_object_throws_S3ObjectTooLargeException() {
         final String bucketName = UUID.randomUUID().toString();
         final String filePath = UUID.randomUUID().toString().concat(".json");
+        final String fileContent = UUID.randomUUID().toString();
 
         final String s3SFile = String.format("s3://%s/%s",bucketName, filePath);
 
-        final GetObjectAttributesResponse getObjectAttributesResponse = GetObjectAttributesResponse.builder().objectSize(8_000_000L).build();
+        final GetObjectRequest getObjectRequest = GetObjectRequest.builder().bucket(bucketName).key(filePath).build();
+        final InputStream fileObjectStream = IOUtils.toInputStream(fileContent, StandardCharsets.UTF_8);
+        final ResponseInputStream<GetObjectResponse> fileInputStream = new ResponseInputStream<>(
+                GetObjectResponse.builder().contentLength(8_000_000L).build(),
```

Comment:
This is another magic number. It could be a constant named `SIZE_EXCEEDING_THRESHOLD` instead.